frontend/src/app/components/fhir-card/common/observation-bar-chart/observation-bar-chart.component.ts

```diff
@@ -15,7 +15,7 @@ const defaultChartEntryHeight = 30;
   styleUrls: ['./observation-bar-chart.component.scss']
 })
 export class ObservationBarChartComponent implements OnInit {
-  @Input() observations: [ObservationModel]
```

Comment:
🤦

Reply:
lol, hey, that 🤦is mine. I'm the one who made this file 😆 

I just find it interesting that it all worked regardless. I only noticed it because I was doing some testing in my other branch that has a lot more change going on and storybook finally complained near the end of my changes.

Reply:
haha, don't be too hard on yourself, I've definitely done stuff like that -- I thought you copy pasted this from my original file :D